Signal/src/ViewControllers/SignalAttachment.swift

```diff
@@ -251,7 +251,12 @@ class SignalAttachment: NSObject {
     // Image attachments may be converted to another image format before 
     // being uploaded.
     private class var inputImageUTISet: Set<String> {
-        return MIMETypeUtil.supportedImageUTITypes().union(animatedImageUTISet)
+         // HEIC is valid input, but not valid output. Non-iOS11 clients do not support it.
+        let heicSet: Set<String> = Set(["public.heic"])
```

Comment:
In my cursory reading of the docs, it seems like there a couple of container formats (.heic and .heif) which can contain a variety of encodings.  Will this single UTI cover them all?

Reply:
In all my testing Photos are captured (static, live, pano) as image.heic. However the scenario you outlined below, receiving a .HEIF from elsewhere and saving it into Photos, does make it possible to import a .HEIF.

I've added the public.heif type and verified you can select such a file now.  Thanks!